tests/41end-to-end-keys/07-backup.pl

```diff
@@ -0,0 +1,387 @@
+my $fixture = local_user_fixture();
+
+my $current_version; # FIXME: is there a better way of passing the backup version between tests?
```

Comment:
I would prefer if each test was a standalone test, rather than there being implicit dependencies (it makes it harder to reason about each test and change later on).

I think once the APIs are split out into separate functions that will reduce boiler plate enough to make it feasible to just start from scratch for each test. There are also `multi_test` if you want to do multiple tests in one run through.